doc/releases/migration-guide-3.7.rst

```diff
@@ -351,6 +351,11 @@ Controller Area Network (CAN)
     :c:func:`can_set_mode` during application startup instead of failing when :c:func:`can_recover`
     is called at a later point in time.
 
+Crypto
+======
+
+* The CSS driver has been deprecated on NXP lpc55s36
```

Comment:
```suggestion
* The CSS driver has been deprecated on NXP lpc55s36 (:github:`71173`).
```

Please add a link to the PR which introduced this change

Reply:
> (:github:`71173`)
Are you requesting this because there is no migration path proposed at this point?